repositories/files/usersdata.csv

```diff
@@ -0,0 +1,4 @@
+1,Gopher,gopher@goolge.com,undefined,1
+2,Rob Pike,go.pike@google.com,male,1
+3,Robert Griesemer,go.griesemer@google.com,male,1
+4,Ken Thompson,go.thompson@google.com,male,1
```

Comment:
The CSV files should have an empty line at the end, because if your app adds a new registry, it will be added in the current line, so you will have two registries in the same line.

Reply:
Resolved :)